### PR TITLE
Grid usage

### PR DESCRIPTION
**Pull request by wohajo**

### CODE REVIEW

File: gtt-jobs/ui/dist/main.css [hunk 1]
```diff
@@ -0,0 +1,7239 @@
+/*
```
Comment (dzkb) [inline gtt-jobs/ui/dist/main.css:1]: I believe this file should not be tracked on git.

File: gtt-jobs/ui/dist/main.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const L = window["L"];
```
Comment (dzkb) [inline gtt-jobs/ui/dist/main.js:1]: I believe this file should not be tracked as well. Both are transpiled distributables.

File: gtt-jobs/ui/index.html [hunk 3]
```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Job market</title>
+    <link rel="stylesheet" href="dist/main.css">
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.7.1/dist/leaflet.css" />
+<script src="https://unpkg.com/leaflet@1.7.1/dist/leaflet.js"></script>
```
Comment (dzkb) [inline gtt-jobs/ui/index.html:10]: In the future we'll have to provide all the dependencies locally. In MTA we cannot reliably request for external assets - for each new domain we'd have to ask the player for permission to make a request.

File: gtt-jobs/ui/_sass/main.scss [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+.job-select-button:hover {
+    background-color: $hover-button-color;
+    cursor: pointer;
+}
+
+// responsive font
+
+@media screen and (max-width: 1110px) and (max-height: 970px), (max-height: 970px) {
+    .jobs, .balance, .filters {
+        font-size: x-large;
```
Comment (dzkb) [inline gtt-jobs/ui/_sass/main.scss:151]: I see you're using media query for managing font size. Is there any option to make the font size relative to the view, like using viewport units + media query (to maintain a minimal font-size for text to be readable)?

**Review by dzkb — CHANGES_REQUESTED** [review]

File: gtt-jobs/ui/dist/main.js (hunk 2 above)
Comment (wohajo) [reply gtt-jobs/ui/dist/main.js]: I am sorry, I misunderstood previous request. This is now fixed and untracked.

File: gtt-jobs/ui/index.html (hunk 3 above)
Comment (wohajo) [reply gtt-jobs/ui/index.html]: Working on importing leaflet to JS from NPM.

File: gtt-jobs/ui/_sass/main.scss (hunk 4 above)
Comment (wohajo) [reply gtt-jobs/ui/_sass/main.scss]: I changed font size to be related to viewport - needs to be reviewed.

**Review by dzkb — APPROVED** [review]

**Pull request merged**